cumulus/parachains/pallets/collective-content/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "pallet-collective-content"
```

Comment:
this is experimental and may undergo numerous changes based on the needs of collectives. this is why it was decided not to integrate it into the frame for now.